### DISCUSSION TITLE
FDS/firesimulation_sprinkler version difference

### DISCUSSION THREAD

**Discussion by saemibang** [firemodels/fds#11421] [Q&A]
I'm trying to simulate a sprinkler pattern. 
I put the same input file using 'TABLE_DATA' in fds version 6.7.5 and version 6.7.9, but the result was different. 
Is there a big difference in droplets between the two versions?
Especially, " &PROP QUANTITY='NUMBER CONCENTRATION' " results

**Comment by drjfloyd** [comment]
There have been some bug fixes to calculating particle drag and movement since 6.7.5.

Is your input just a sprinkler or is there also a fire or other flow inputs? Is it slightly different or very different? 

If your input is more than just a sprinkler disharging, I'd suggest changing the inputs to be just a sprinkler and no fire or other flows and see if you still see the diffrence. 

If your input is just a sprinkler discharging and the results are very different, please open an issue and attach your input file.

**Reply by saemibang** [reply]
Thank you for your prompt reply.

This is an input file.

&PART ID='water drops', SPEC_ID='WATER VAPOR', DISTRIBUTION=ROSIN-RAMMLER-LOGNORMAL,CHECK_DISTRIBUTION=.TRUE.,
      QUANTITIES(1:2)='PARTICLE DIAMETER','PARTICLE TEMPERATURE',COLOR='BLUE',
      DIAMETER=3000.0, INITIAL_TEMPERATURE=15.0, SAMPLING_FACTOR=1, BREAKUP=.FALSE., SURFACE_TENSION=0.072 /
&PROP ID='K-11', PART_ID='water drops',QUANTITY='SPRINKLER LINK TEMPERATURE', OFFSET=0.100,
      PARTICLE_VELOCITY=14.14,
      SPRAY_PATTERN_SHAPE='GAUSSIAN', SPRAY_PATTERN_TABLE='TABLE1',
      FLOW_RATE=20., SMOKEVIEW_ID='sprinkler_pendent', PARTICLES_PER_SECOND=10000 /

&TABL ID='TABLE1', TABLE_DATA= 23.3031744317158, 27.3850552526895, 180, 185,14.14, 0.000470211946496468 /
&TABL ID='TABLE1', TABLE_DATA= 27.3850552526895, 31.2131535856347, 180, 185,14.14, 0.000229133304181882 /
&TABL ID='TABLE1', TABLE_DATA= 31.2131535856347, 34.7835177406385, 180, 185,14.14, 0.000372839341717754 /
&TABL ID='TABLE1', TABLE_DATA= 34.7835177406385, 38.1001051371916, 180, 185,14.14, 0.00070549890971641 /
&TABL ID='TABLE1', TABLE_DATA= 38.1001051371916, 41.1725632386487, 180, 185,14.14, 0.00250526318837726 /
&TABL ID='TABLE1', TABLE_DATA= 41.1725632386487, 44.0142534207921, 180, 185,14.14, 0.000794545975796572 /
&TABL ID='TABLE1', TABLE_DATA= 44.0142534207921, 46.6406535654996, 180, 185,14.14, 0.000355102324490404 /
&TABL ID='TABLE1', TABLE_DATA= 46.6406535654996, 49.068146827965, 180, 185,14.14, 0.000215378066332101 /
&TABL ID='TABLE1', TABLE_DATA= 49.068146827965, 51.3131644918371, 180, 185,14.14, 0.000525956857782424 /
.
.
.
#fraction The sum of the total fraction is 1.

&PROP ID='PDPA_prop1', PDPA_RADIUS=0.05, PDPA_START=0.0, QUANTITY='DIAMETER', 
      PART_ID='water drops', PDPA_M=3, PDPA_N=0, PDPA_INTEGRATE=.TRUE. /
&PROP ID='PDPA_prop2', PDPA_RADIUS=0.05, PDPA_START=0.0, QUANTITY='W-VELOCITY', 
      PART_ID='water drops', PDPA_M=3, PDPA_N=3, PDPA_INTEGRATE=.TRUE. /
&PROP ID='PDPA_prop3', PDPA_RADIUS=0.05, PDPA_START=0.0, QUANTITY='NUMBER CONCENTRATION', 
      PART_ID='water drops', PDPA_INTEGRATE=.TRUE. /

#PDPA was used to calculate the water flux (LPM/m2)

![image](https://user-images.githubusercontent.com/112543935/217396814-b9a04c68-957e-4a8a-969c-b37ef5b8672e.png)
calculated using version 6.7.9.
![image](https://user-images.githubusercontent.com/112543935/217396938-1a81c70f-32a5-4ca7-950f-1192d49155cf.png)
calculated using version 6.7.5.
Water flux loss occurs in the part exceeding 1 meter along the yyy-axis. 
I  think this to be PDPA NUMBER CONCENTRATION related.

**Comment by drjfloyd** [comment]
I converted this to an issue: 
https://github.com/firemodels/fds/issues/11431

Please go to the issue and attach your full input file. You have only given some of the input file here.  You will need to change the .fds extension to .txt in order to attach the file to the issue.
